test/unit_tests/braket/aws/test_aws_device.py

```diff
@@ -238,6 +238,36 @@ def test_gate_model_sim_schema():
 )
 
 
+@pytest.fixture
+def quantum_task(aws_session, s3_destination_folder):
+    theta = FreeParameter("theta")
+    circ = Circuit().ry(angle=theta, target=0)
+    return AwsQuantumTask.create(
+        device_arn="arn:aws:braket:::device/quantum-simulator/amazon/sv1",
+        aws_session=aws_session,
+        poll_timeout_seconds=2,
+        task_specification=circ,
+        shots=10,
+        s3_destination_folder=s3_destination_folder,
+    )
+
+
+@pytest.fixture
+def quantum_task_batch(aws_session, s3_destination_folder):
```

Comment:
can you call this parameterized_task_batch?

Reply:
Won't this always fail?

Reply:
Yes, for now. Adding in description around the changes for the subsequent PR